scripts/scripts.js

```diff
@@ -155,25 +137,38 @@ export function decorateBlock(block) {
   block.classList.add('block');
   block.setAttribute('data-block-name', shortBlockName);
   block.setAttribute('data-block-status', 'initialized');
+
+  const blockWrapper = block.parentElement;
+  blockWrapper.classList.add(`${shortBlockName}-wrapper`);
 }
 
 /**
  * Decorates all sections in a container element.
  * @param {Element} $main The container element
  */
 export function decorateSections($main) {
-  wrapSections($main.querySelectorAll(':scope > div'));
-  $main.querySelectorAll(':scope > div.section-wrapper').forEach((section) => {
+  $main.querySelectorAll(':scope > div').forEach((section) => {
```

Comment:
Should we take this opportunity to get rid of the `$` notation for good?

Reply:
let's do that separately... it is used in a lot of places...